docs/install.md

```diff
@@ -1,51 +1,73 @@
 # Installing Open Angel Arena
-You have to set up several things before you can run a local instance of AAA for development.
+You have to set up several things before you can run a local instance of OAA for development.
 
 ## System Dependencies
 Start by installing these programs:
  * [Steam](http://steampowered.com)
  * [nodejs](http://nodejs.org)
  * [Github client](http://desktop.github.com/)
- * [Sublime Text 3](http://www.sublimetext.com/) **optional**
+ * [Visual Studio Code](http://code.visualstudio.com/) `Text editor which includes support for github, editorconfig and much more`
 
 Install the Dota 2 Workshop Tools.
  * From within steam, right click on `Dota 2` and select `Properties` at the bottom
  * Go to the DLC tab of the window that pops up
  * Check the `INSTALL` box next to `Dota 2 Workshop Tools DLC`
 
-## Addon Tool
+## Setting evrything up
 Open the `node.js command prompt`. Not `Node.js` itself, but the command prompt.
 
-### Install Node Dependencies
-Run the command 
+### Folder definitions
+Now we will define 3 paths 
+ * `Dota2 installation folder` is something like `C:\Program Files (x86)\Steam\steamapps\common\dota 2 beta\` or at ant other drive or install location, but certainly ends with `dota 2 beta`
+ * `Dota2 addons folder` is folder named `addons` in `Intallation folder`, so it looks something like `\dota 2 beta\addons`
+ * `Open Angel arena folder` is a folder name `oaa` inside of `addons folder`, it contains all the files which are present in this github repo 
+
+**`Dota2 addons folder` and `Open Angel arena folder` may not exist for the first time, they will be created after `Prepare Addon Directory` step**
+
+Final structure sholud look like `C:\(some path to steam library)\Steam\steamapps\common\dota 2 beta\addons\oaa\`
+
+### Navigating with command prompt
+Use command `cd` and the undergoing path to navigate,(`cd` works relative to your current path), for example
+```
+dota 2 beta\addons> cd oaa
+```
+will go from `Addons` folder to `Open Angel Arena` folder, and
+```
+dota 2 beta\addons\oaa> cd ../
```

Comment:
overly complex instructions with more than 2 redundant steps which rely on future steps. Also this has nothing to do with gulp.